AUTHORS

```diff
@@ -1,20 +1,21 @@
 Authors and Contributors
 ========================
 
+- P. Courtier (ECMWF)
```

Comment:
In case you wonder - I just sorted this file when adding myself to the contributors because the file tells us it should be sorted.